qabelbox/src/main/java/de/qabel/qabelbox/config/AppPreference.java

```diff
@@ -0,0 +1,67 @@
+package de.qabel.qabelbox.config;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+/**
+ * Created by danny on 27.01.16.
+ */
+public class AppPreference {
+
+    private final SharedPreferences settings;
+    private final String P_TOKEN = "token";
+    private final String P_LAST_APP_START_VERSION = "lastappstartversion";
+    private final String P_LAST_APP_UPDATE_QUESTION_TIME = "lastupdatequestiontime";
+    private final long NEW_UPATE_QUESTION_TIME_INTERVAL = 1000 * 60 * 60 * 24 * 3l;
+
+    public AppPreference(Context context) {
+
+        settings = context.getSharedPreferences("settings", Context.MODE_PRIVATE);
+    }
+
+    public void setToken(String token) {
+
+        settings.edit().putString(P_TOKEN, encrypt(token)).commit();
+    }
+
+    private String encrypt(String data) {
+        //@todo add functions to encrypt
```

Comment:
I don't think we need to protect the token any more than our private key, at least not until the KeyStore is integrated.